components/dex.libsonnet

```diff
@@ -6,13 +6,20 @@
     image: error 'must provide image',
     version: error 'must provide version',
     namespace: error 'must provide namespace',
+    tlsSecret: '%s-tls' % [dex.config.name],
+    tlsCertMapEntry: 'tls.crt',
```

Comment:
Nit: can we call these: `tlsCertKey` and `tlsKeyKey` respectively? These fields do not belong to a ConfigMap, and thus having `Map` in the name is confusing. Additionally, using `Entry` sounds awkward to me since we don't use this terminology anywhere. They double `Key` is clunky but it is at least consistent with our other repos: https://github.com/observatorium/observatorium/blob/master/jsonnet/lib/observatorium-api.libsonnet#L222

Reply:
Alright, I used "Entry" because `KeyKey` sounded really wrong, but I agree with your assessment.